docs/reference/intro.mdx

```diff
@@ -0,0 +1,51 @@
+
+Given all these requirements (along with others such as an admin UI, audit trails, environment management, and more) it's clear that full use of feature flags requires a feature flagging system - typically a stand-alone feature-flagging service along with a client library which interacts with that service.
+
+<figure>
+<img src={require('@site/static/img/docs/reference/intro/ff-service.png').default}/>
+<figcaption><i>A typical feature-flagging system</i></figcaption>
+</figure>
+
+## What is OpenFeature?
+
+OpenFeature provides a shared, standardized feature flagging client - an *SDK* - which can be plugged in to various 3rd-party feature flagging *providers*. Whether you're using an open-source system or a commercial product, whether it's self-hosted or cloud-hosted, OpenFeature provides a consistent, unified API for developers to use feature flagging in their applications. 
+
+<figure>
+<img src={require('@site/static/img/docs/reference/intro/of-architecture.png').default}/>
+<figcaption><i>OpenFeature integrated with a hypothetical "Flags-R-us" flag management system</i></figcaption>
+</figure>
+
+To accomplish this, the OpenFeature SDK defines a number of flexible abstractions.
+
+### Evaluation API
+
+The [evaluation API](./concepts/01-evaluation-api.mdx) is the part of the OpenFeature SDK that an _application author_ interacts with. It allows developers to evaluate feature flags, and use the resulting values to impact control flow or application characteristics. The evaluation API provides a framework that allows for customization of behavior and integration with various tools.
+
+### Evaluation Context
+
+The [evaluation context](./concepts/03-evaluation-context.mdx) is a container for arbitrary contextual data that can be used as a basis for dynamic evaluation. Static data such as the host or an identifier for the application can be configured globally. Dynamic evaluation context, such as the IP address of the client in a web application, can be implicitly propagated or explicitly passed to during flag evaluation, and can be merged with static values.
+
+### Providers
+
+[Providers](./concepts/02-provider.mdx) are the "translation layer" between the evaluation API and the flag management system in use. Providers are responsible for mapping the arguments supplied to the evaluation API to their equivalent representation in the associated flag management system. Providers might wrap a vendor SDK, call a bespoke flag evaluation REST API, or even parse some locally stored file to resolve flag values.
+
+### Hooks
+
+[Hooks](./concepts/04-hooks.mdx) are a mechanism that allow for the addition of arbitrary behavior at various points in the _flag evaluation life-cycle_. Hooks let you extend the OpenFeature SDK, adding functionality such as validating a resolved flag value, modifying or adding data to the evaluation context, logging, telemetry, and tracking.
```

Comment:
```suggestion
[Hooks](./concepts/04-hooks.mdx) are a mechanism that allows for the addition of arbitrary behavior at various points in the _flag evaluation life-cycle_. Hooks let you extend the OpenFeature SDK, adding functionality such as validating a resolved flag value, modifying or adding data to the evaluation context, logging, telemetry, and tracking.
```

Reply:
@moredip this is a great change that I would like to get in as soon as possible. When you have a moment, would you be able to address the comments above?